clarity/evaluator/haspi/ip.py

```diff
@@ -210,7 +210,7 @@ def get_neural_net():
 
 def nn_feed_forward_ensemble(
     data: np.ndarray, neural_net_params: Dict, weights_hidden, weights_out
-) -> float:
+) -> np.ndarray:
```

Comment:
This is an error. The nn returns an nd.array containing the activations of the output layer